### DISCUSSION TITLE
Where the fuck is rightJoin() ???

### DISCUSSION THREAD

**Discussion by MichaelBelgium** [doctrine/doctrine2#10694] [Support Questions]
Worst PHP ORM really .. 👎 

Am i the only one who does right joins? doctrine devs forgot this exist?
 [QueryBuilder ](https://github.com/doctrine/orm/blob/2.15.x/lib/Doctrine/ORM/QueryBuilder.php) is missing rightJoin, is there any other way to do a right join?

**Comment by derrabus** [comment]
Sorry, this is not the way we talk to each other on this discussion board.
